### ISSUE TITLE
Sliding in Firefox flickers

### ISSUE THREAD

**Issue by sir-gawain** [artpolikarpov/fotorama#352]
Sliding the gallery using the mouse flickers in Firefox 35 (on Mac OS 10.10.1 and others) if sliding to the right revealing the left image. As long as the motion is not finished, the image is hidden. Stopping the motion makes the image visible. This happens also in the gallery on http://fotorama.io/.
Adding 
```
.fotorama__wrap--slide .fotorama__stage__frame {
   opacity: 1 !important;
   perspective: 1px; /* this was added */
}
```
fixes this problem.
I'm not sure, if that breaks other browsers.

**Comment by KODerFunk** [comment]
@sir-gawain thanks :thumbsup: 
I fix it localy, but someone must PR this.

**Comment by KODerFunk** [comment]
It fix in one place, but causes an error elsewhere. http://take.ms/emTJP

**Comment by KODerFunk** [comment]
Fixed in 35.0.1! Can be closed.

**Comment by sir-gawain** [comment]
Yes, seems to be fixed in FF.
